lute/luau/src/luau.cpp

```diff
@@ -2574,6 +2579,14 @@ int luau_parse(lua_State* L)
     lua_pushnumber(L, result.parseResult.lines);
     lua_setfield(L, -2, "lines");
 
+    lua_createtable(L, serializer.lineOffsets.size(), 0);
+    for (size_t i = 0; i < serializer.lineOffsets.size(); i++)
+    {
+        lua_pushnumber(L, serializer.lineOffsets[i]);
+        lua_rawseti(L, -2, i + 1);
```

Comment:
Are you sure you need to use rawseti here?

If this is for arrays, you should be able to just use `lua_setfield`.
You can see how `system.cpus` does it here: https://github.com/luau-lang/lute/blob/primary/lute/system/src/system.cpp